lib/msf/ui/console/module_option_tab_completion.rb

```diff
@@ -122,6 +123,10 @@ def tab_complete_option_names(mod, str, words)
         # Provide tab completion for option values
         #
         def tab_complete_option_values(mod, str, words, opt:)
+          if words.last.downcase == 'sessiontlvlogging'
```

Comment:
```suggestion
          if words.last.casecmp?('sessiontlvlogging')
```

As it avoid the intermediate string assignment